tests/std/tests/P0024R2_parallel_algorithms_transform_reduce/test.cpp

```diff
@@ -144,6 +146,25 @@ void test_case_incorrect_special_case_reasoning() {
     assert(transform_reduce(begin(c), end(c), d, 0, multiplies<>{}, plus<unsigned char>{}) == 0);
 }
 
+void test_case_narrowing_conversion() {
+    size_t a[]         = {1, 2, 3};
+    auto return_itself = [](size_t a) { return a; };
+    // Initializing a smaller type (int here) with a larger type (size_t here).
+    // According to [transform.reduce](7.1-7.4),

```

Comment:
These Standardese citations are great! While section names like \[transform.reduce\] are stable, paragraph numbers aren't, so we conventionally cite the Working Paper number when mentioning paragraph numbers, so we can track down what was meant if the paragraphs are ever reshuffled. N4971 is the current WP, and these numbers are still valid. Occurs below.

Also, the "*Mandates:* All of \[...\] are convertible to `T`." part is important, so I think we should cite paragraphs 7 and 3 in their entirety, not just their sub-bullets.

Finally, our style is to say \[stable.name\]/paragraph.